### PR TITLE
Increment minor version of app and chart

### PR DESCRIPTION
**Pull request by tkerdoncuff**

### CODE REVIEW

File: helm/influxdb-athena-crawler/README.md [hunk 1]
```diff
@@ -56,15 +56,18 @@ helm install influxdb-athena-crawler influxdb-athena-crawler/influxdb-athena-cra
...
 | defaults.cleanObjects | bool | `false` | Whether to delete S3 objects after processing them. |
+| defaults.maxObjectAge | string | `"5m"` | After how long to delete the objects. |
 | defaults.timeout | string | `"10m"` | The global timeout. |
 | defaults.influxServers | list | `[]` | The InfluxDB servers addresses. |
 | defaults.influxToken | string | `""` | The InfluxDB token. |
 | defaults.influxOrg | string | `""` | The InfluxDB org to write to. |
 | defaults.influxBucket | string | `""` | The InfluxDB bucket write to. |
 | defaults.measurement | string | `""` | The InfluxDB bucket measurement. |
 | defaults.timestampRow | string | `"timestamp"` | The timestamp row in CSV. |
-| defaults.timestampLayout | string | `"2006-01-02T15:04:05.000Z"` | The layout to parse timestamp. |
```
Comment (antonincms) [inline helm/influxdb-athena-crawler/README.md:67]: Is that related ?

Comment (tkerdoncuff) [reply helm/influxdb-athena-crawler/README.md:67]: Good catch, it is intended yes, looks like I failed to regenerate the doc during the work on #332 
The idea is to have the default value match the format usually present in the CSVs.

**Review by antonincms — APPROVED** [review]

File: helm/influxdb-athena-crawler/README.md (hunk 1 above)
Comment (antonincms) [reply helm/influxdb-athena-crawler/README.md:67]: Ok !
